packages/chat/src/components/AgentMessage/AgentMessage.css.ts

```diff
@@ -0,0 +1,50 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+import { COLORS } from '@/styles/colors';
+import { FAMILY } from '@/styles/font';
+
+export const messageContainer = recipe({
+  base: {
+    backgroundColor: COLORS.NEUTRAL_LIGHT[50],
+    color: COLORS.NEUTRAL_DARK[900],
+    padding: '11px 16px 10px',
+    fontFamily: FAMILY,
+    position: 'relative',
+    fontSize: '14px',
+    lineHeight: '20px',
+    borderRadius: '10px',
+  },
+  variants: {
+    isCodeBlock: {
+      true: {
+        padding: 0,
+      },
+    },
+    generated: {
+      true: {
+        borderBottomLeftRadius: 0,
+        borderBottomRightRadius: 0,
+      },
+    },
+  },
+});
+
+export const generatedChin = style({
+  display: 'flex',
+  alignItems: 'center',
+  padding: '10px 16px 9px',
+  backgroundColor: COLORS.NEUTRAL_LIGHT[100],
+  color: COLORS.NEUTRAL_DARK[200],
+  fontFamily: FAMILY,
+  fontSize: '12px',
+  whiteSpace: 'nowrap',
+  lineHeight: '17px',
+  gap: 10,
```

Comment:
is this supposed to be `10px` ?  
I think in `vanilla-extract` the 'px' is the default unit, so we don't need to place units in most places.  
I realized you did though, so if you prefer it with units, then lets at least be consistent. :)